website/docs/governance/organisation-and-roles.md

```diff
@@ -103,7 +103,7 @@ The objective of this organisation is to be very inclusive, evolutive and open t
 - Define detailed functional specifications
```

Comment:
- Remplacer
- ~~Prioritize the different functionalities to be developed.~~
```
- Define detailed functional specifications
```
- par
```suggestion
- Inform the SemApps team and in particular DevX role of priorities when they are fund
- Define detailed functional specifications
```

Reply:
Pense à retirer la ligne 102

Reply:
Hmm je crois qu'on avait parlé du rôle DevX à un moment mais je ne vois pas/plus pourquoi c'est ce rôle qui devrait être informé. C'est plutôt le rôle Développement non ?

Reply:
Oups, mauvaise prise de note en direct de ma part. Tu as raison rôle Developpement